Makefile

```diff
@@ -208,15 +222,8 @@ centos7_docker_publish: Dockerfile-centos7.$(PLATFORM).tag
 
```

Comment:
`DT_LARGE_TESTS_ROOT` env will be passed for centos7_in_docker, centos7_build_in_docker_impl, that's the only difference I can see.